style.css

```diff
@@ -674,3 +691,160 @@
   order: 0;
   flex-grow: 0;
 }
+
+.contact-form {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 120px 24px;
+  gap: 58px;
+  position: absolute;
+  width: 375px;
+  height: auto;
+  left: 0;
+  top: 6238px;
+}
+
+.contact-intro-cont {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 0;
+  gap: 24px;
+  width: 327px;
+  height: 216px;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.contact-intro-cont p {
+  width: 327px;
+  height: 216px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  text-align: center;
+  color: #172b4d;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.the-form {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 0;
+  gap: 24px;
+  width: 327px;
+  height: auto;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+  text-align: center;
+}

```

Comment:
- I think your form is not properly designed. Can you make it look almost identical to what we have on Figma? 

Here is what I can currently see, compared to what's on Figma 👇🏽 
| Your Design  | Figma Design |
| :---:   | :---: | 
|![bad](https://user-images.githubusercontent.com/99246359/200664511-ba7243f0-fe45-4e22-964f-bdf4320be77b.PNG)  |![gdfigma](https://user-images.githubusercontent.com/99246359/200664651-b844677b-7510-46f8-85b6-cf564e08e1aa.PNG)  |  

- I think you can start by using `border-bottom: 1px solid #dbd8d7;` for each input, according to figma, you just need a bottom border and some extra styling 🌸

Reply:
Thanks. Fixed.